manifests/g/G0rocks/Cab/V0.0.0-Alpha/G0rocks.Cab.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: G0rocks.Cab
+PackageVersion: V0.0.0-Alpha
+InstallerType: portable
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 787.)